test/Carter.Tests/Carter.Tests.csproj

```diff
@@ -8,4 +8,7 @@
   <ItemGroup>
     <ProjectReference Include="..\..\src\Carter\Carter.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="MartinCostello.Logging.XUnit" Version="0.1.0" />
```

Comment:
I think if we add this we need to have a test to check logs but then i'm not sure we want to have unit tests for logs?